liberum_core/src/node/store.rs

```diff
@@ -230,16 +284,19 @@ mod tests {
             .await
             .unwrap();
         let node_store = kameo::spawn(node_store);
-        let new_node = Node::builder()
-            .name("test_node".to_string())
-            .keypair(Keypair::generate_ed25519())
-            .build()
-            .unwrap();
+        let node_snapshot = NodeSnapshot {
```

Comment:
Couldn't an alternative method to `build` be added to the builder, which could be used to build the snapshots? Most of the fields are the same.